charts/csm-authorization/values.yaml

```diff
@@ -0,0 +1,93 @@
+# if enabled, nginx ingress controller will be deployed
+# if you have your own ingress controller, keep this false and set the appropriate annotations for the ingresses in the authorization section
+ingress-nginx:
+  enabled: true
+
+# if enabled, cert-manager will be deployed
+# if cert-manager is already deployed, keep this false
+cert-manager:
+  enabled: true
+
+authorization:
+  # images to use in installation
+  images:
+    proxyService: dellemc/csm-authorization-proxy:v1.5.0
+    tenantService: dellemc/csm-authorization-tenant:v1.5.0
+    roleService: dellemc/csm-authorization-role:v1.5.0
+    storageService: dellemc/csm-authorization-storage:v1.5.0
+    opa: openpolicyagent/opa
+    opaKubeMgmt: openpolicyagent/kube-mgmt:0.11
+
+  # base hostname for the ingress rules that expose the services
+  # the proxy-server ingress will use this hostname
+  # the tenant-service ingress will use tenant.hostname
+  # the role-service ingress will use role.hostname
+  hostname: csm-authorization.com
+
+  # log level for csm-authorization
+  logLevel: debug
+
```

Comment:
For Authorization, there has been a new parameter added https://github.com/dell/helm-charts/blob/release-1.5.1/charts/csm-authorization/values.yaml#L30.

Reply:
Thanks for reviewing, updated the charts with latest changes.